app/validators/intersection-task.js

```diff
@@ -2,11 +2,15 @@ export default function validateIntersectionTask() {
   return (key, newValue, oldValue, changes, content) => {
     const customerChanged =
       Object.keys(changes).includes("customer") &&
```

Comment:
Not need with the following [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) operator.

Reply:
I have to correct myself, this line is indeed necessary.